networkx/generators/classic.py

```diff
@@ -681,16 +681,16 @@ def trivial_graph(create_using=None):
     return G
 
 
-def turan_graph(n,r):
+def turan_graph(n, r):
     """ Return the Turan Graph
     
     The Turan Graph is a complete multipartite graph on `n` vertices
     with `r` partitions with the property that it has the maximum number
     of edges for any graph with the same number of vertices and partitions.
 
-    Given n and r, we get a complete multipartite graph with `r-(n mod r)`
-    partitions of size `n/r`, rounded down, and `n mod r` partitions of size
-    `n/r+1`, rounded down.
+    Given `n` and `r`, we generate a complete multipartite graph with
+    :math:`r-(n \mod r)` partitions of size :math:`n/r`, rounded down, and
+    :math:`n \mod r` partitions of size :math:`n/r+1`, rounded down.
```

Comment:
If using a backslash here, you'll need to prepend an `r` to the initial triple quotes of this docstring, like this: `r"""'`. This prevents having to escape the backslash in the Python string.